### ISSUE TITLE
KubernetesHelmTask: Implement `k8s:helm` equivalent functionality `k8sHelm` for Kubernetes Gradle Plugin

### ISSUE THREAD

**Issue by manusa** [eclipse/jkube#1052]
### Description

Child of #31 
Blocked by #1029

Port HelmMojo from maven plugins to KubernetesHelmTask
- Any functionality present in maven plugin goal should be moved to KubernetesHelmTask

## AC
- [ ] Chart can be customized through properties (Configuration should be added to KubernetesExtension)
- [ ] Chart can be customized though DSL (Configuration should be added to KubernetesExtension)
- [ ] If no equivalents can be found for the extra-info in Gradle (Maintainers, etc.), create a new issue for the future
- [ ] doc: Create appropriate asciidoc entry
